services/nublado/values-minikube.yaml

```diff
@@ -0,0 +1,99 @@
+nublado:
```

Comment:
I'm confused -- this seems to be a configuration for the existing nublado, not for your new work, but we won't ever run the existing nublado in this environment, will we?

Reply:
Ah this is a good point, but then I remember the TLS for cert manager is somehow knotted around nublado I think due to the cert being put in that namespace.  So we needed this I think to get the cert to be requested.